test/browser/custom-elements.js

```diff
@@ -1,870 +0,0 @@
-      DOM: makeDOMDriver(createRenderTarget(), {
-        'my-element': myElementDef
-      })
-    });
-    // Make assertions
-    responses.DOM.select(':root').observable.skip(1).take(1).subscribe(function (root) {
-      let myElement = root.querySelector('.inner-element');
-      assert.notStrictEqual(myElement, null);
-      assert.notStrictEqual(typeof myElement, 'undefined');
-      assert.strictEqual(myElement.tagName, 'H3');
-      assert.strictEqual(myElement.textContent, 'Hello world');
-      assert.strictEqual(myElement.style.color, 'rgb(255, 0, 0)');
-      responses.dispose();
-      done();
-    });
-  });
-
-  it('should throw if properties driver getter has no args', function (done) {
-    // Make custom element
-    function myElementDef(ext) {
-      return {
-        DOM: ext.props.get().map(propsObj =>
-          h('h3.inner-element',
-            {style: {color: propsObj.color}},
-            String(propsObj.content)
-          )
-        )
-      };
-    }
-    function app() {
-      return {
-        DOM: Rx.Observable.just(
-          h('div', [
-            h('my-element', {color: '#FF0000', content: 'Hello world'})
-          ])
-        )
-      };
-    }
-    let [requests, responses] = Cycle.run(app, {
-      DOM: makeDOMDriver(createRenderTarget(), {
-        'my-element': myElementDef
-      })
-    });
-    // Make assertions
-    responses.DOM.select(':root').observable.skip(1).take(1).subscribe(
-      function onNextHandler() {
-        assert.fail('DOM :root Observable should not emit onNext.');
-      },
-      function onErrorHandler(err) {
-        assert.strictEqual(err.message, 'Custom element driver `props.get()` ' +
-          'expects an argument in the getter.'
-        );
-        done();
-      }
-    );
-  });
-
-  it('should allow nested svg elements as children', function (done) {
-    // Make the svg custom element
-    function workspace(responses) {
-      return {
-        DOM: responses.props.get('*').map(props => {
-          const svgProps = {...props, children: void 0};
-          return svg('svg', {
-            attributes: svgProps,
-            style: {border: '1px solid rgb(221, 221, 221)'}},
-            props.children
-          );
-        })
-      }
-    }
-    // Use the custom elements
-    function app() {
-      return {
-        DOM: Rx.Observable.just(
-          h('section', [
-            h('work-space', {width: 500, height: 500}, [
-              h('work-space', {width: 100, height: 100})
-            ])
-          ])
-        )
-      };
-    }
-    let [requests, responses] = Cycle.run(app, {
-      DOM: makeDOMDriver(createRenderTarget(), {
-        'work-space': workspace,
-      })
-    });
-    // Make assertions
-    responses.DOM.select(':root').observable.skip(1).take(1).subscribe(function (root) {
-      let svgElements = root.querySelectorAll('svg');
-      assert.strictEqual(svgElements.length, 2);
-      responses.dispose();
-      done();
-    });
-  })
```

Comment:
Would you like me to add this test as a nested dialogue SVG for *render-dom,js*?